src/DependencyInjection/Security/Factory/ConnectFactory.php

```diff
@@ -20,7 +21,7 @@
 /**
  * @author Marc Weistroff <marc.weistroff@sensiolabs.com>
  */
-class ConnectFactory extends AbstractFactory
+class ConnectFactory extends AbstractFactory implements AuthenticatorFactoryInterface
```

Comment:
As the authenticator also provides an entry point, you must implement `EntryPointFactoryInterface` in this factory.

If this bundle still allows <5.1 (I think it does, as `symfonycorp/connect` allows it), you should create a new class that extends this one and implements both new interfaces. See https://github.com/scheb/2fa/blob/master/src/bundle/SchebTwoFactorBundle.php#L39-L44 for an example.

Reply:
Both should be fixed now.